### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RenanPalmeira**
Added projects, drivers and topic 'Projects'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,4 +336,8 @@
 
 ## Project Management
   
-  * [milestones](https://github.com/turbopape/milestones) 
+  * [milestones](https://github.com/turbopape/milestones)
+ 
+## Products
+
+  * [Metabase](http://www.metabase.com/)
```
Comment (razum2um) [inline README.md]: @RenanPalmeira please, change link to https://github.com/metabase/metabase and put this to the top under "Awesome products in Clojure"

Comment (RenanPalmeira) [inline README.md]: Fixed :+1:

Comment (razum2um) [inline README.md]: seems, you got me wrong, https://github.com/razum2um/awesome-clojure/blame/master/README.md#L3 such section already exits :)

Comment (razum2um) [inline README.md]: @RenanPalmeira ooh, seems I also lost the track of this :D 
metabase already listed https://github.com/razum2um/awesome-clojure/commit/042a6ad0a16b1ed51dca1fc231990bc16a9b02f9 sorry for misleading comment

Comment (RenanPalmeira) [inline README.md]: @razum2um hahaha ok, I have removed :smile:

**Pull request merged**